contrib/cscope/extensions.el

```diff
@@ -0,0 +1,15 @@
+;;; extensions.el --- cscope Layer extensions File for Spacemacs
```

Comment:
I think current convention is not to include this file in contributed layers if it is empty. IMO it shouldn't even be part of the layer template, it is infrequently used.